demos/zsaw_mmc3/prg/mmc3.cfg

```diff
@@ -22,7 +22,7 @@ SEGMENTS {
    RAM:        load = RAM,                 type = bss, start = $0300;
    HEADER:     load = HDR,                 type = ro,  align = $10;
    MUSIC_0:    load = PRG0,                type = ro;
-   MUSIC_1:    load = PRG1,                type = ro;
+   MUSIC_1:    load = PRG0,                type = ro;
```

Comment:
For testing the address changes, i moved MUSIC_1 into prg 0 so it starts at a different address than music 0